template/{{.project_name}}/bundle/targets.yml.tmpl

```diff
@@ -23,4 +25,4 @@ targets:
     # We use 'mode: production' to indicate this is a production deployment.
     mode: production
     workspace:
-      root_path: /Shared/{{.project_name}}/
+      root_path: /Workspace/Users/${workspace.current_user.userName}/.bundle/${bundle.name}/my-envs/${bundle.target}
```

Comment:
why make it this complicated? I put it in the shared with the reason that it is easy to find when deployed by an SP